source/manuals/grafana-dashboards.html.md.erb

```diff
@@ -1,58 +1,61 @@
+The [GDS Application Metrics Default Dashboard](https://grafana-paas.cloudapps.digital/d/000000011/gds-default-dashboard?orgId=1) displays application metrics produced by  [Ruby](https://github.com/alphagov/gds_metrics_ruby), [Java with Dropwizard](https://github.com/alphagov/gds_metrics_dropwizard) and [Python](https://github.com/alphagov/gds_metrics_python) clients.
+
+If you’ve configured your application with one of these libraries you can view its metrics. You can do this by selecting your application from the **Available Apps** dropdown:
 
 ![Available Apps](images/available-apps.png)
 
-If you can't see your application in the list, make sure you've followed the instructions in [Setting up metrics](../manuals/monitor-paas-app-with-prometheus.html) and that your changes have been deployed to the [GOV.UK PaaS](https://docs.cloud.service.gov.uk/).
+If your application is not displayed, check the instructions for [setting up metrics](../manuals/monitor-paas-app-with-prometheus.html) and that any changes are deployed to [GOV.UK Platform as a Service (PaaS)](https://docs.cloud.service.gov.uk/).
 
-[See a full explanation of what this default dashboard shows](../manuals/gds-application-metrics-default-dashboard.html)
+You can see a full explanation of [what the default dashboard shows here](../manuals/gds-application-metrics-default-dashboard.html)
 
 ### GDS Container Metrics Default Dashboard
 
-The [GDS Container Metrics Default Dashboard](https://grafana-paas.cloudapps.digital/d/E2dUvczmz/container-metrics?orgId=1) shows container metrics produced by the [Paas Metric Exporter](https://github.com/alphagov/paas-metric-exporter). If you are running the PaaS metric exporter then you can view metrics for your application by selecting your application from the 'App' dropdown.
+The [GDS Container Metrics Default Dashboard](https://grafana-paas.cloudapps.digital/d/E2dUvczmz/container-metrics?orgId=1) displays container metrics produced by the [paas-metric-exporter](https://github.com/alphagov/paas-metric-exporter).
+
+If you’re running the PaaS metric exporter you can view metrics for your application by selecting it from the **App** dropdown.
 
-If you can't see your application in the list, make sure you've followed the instructions in [Set up the PaaS metric exporter app with Prometheus](../manuals/set-up-paas-metric-exporter-with-prometheus.html) and that your changes have been deployed to the [GOV.UK PaaS](https://docs.cloud.service.gov.uk/).
+If your application is not displayed, check the instructions for [setting up the PaaS metric exporter app with Prometheus](../manuals/set-up-paas-metric-exporter-with-prometheus.html) and that any changes are deployed to GOV.UK PaaS.
 
 ### Official and community built dashboards
 
 You can [import dashboards](http://docs.grafana.org/reference/export_import/#importing-a-dashboard) from the [official and community built dashboards](https://grafana.com/dashboards) on the Grafana website.
```

Comment:
It would be good to say what the benefits of this would be and if there's any we think could be particularly useful to people.